clickhouse-http-client/src/test/java/com/clickhouse/client/http/ApacheHttpConnectionImplTest.java

```diff
@@ -92,6 +88,7 @@ public void testConnection() throws Exception {
     }
 
     @Test(groups = { "integration" })
+    @Ignore("Need to disable the option to provide custom socket factory. note: need to remove it")
```

Comment:
Why would we need to remove it? 
It is a bit strange to provide custom socket factory, but it was added for some reason. Certificates?